test/job.js

```diff
@@ -46,13 +47,34 @@ describe('/POST Post a new job', () => {
           response.body.should.have
             .property('message')
             .eql('Create the job successful!');
-          response.body.job.should.have.property('id');
-          response.body.job.should.have
-            .property('title')
-            .eql(jobParameters.title);
-          response.body.job.should.have
-            .property('tags')
-            .eql(jobParameters.tags);
+          id = response.body.job.id;
+          done();
+        });
+    });
+  });
+
+  describe('/PUT Update the job has just been create', () => {
```

Comment:
- The test case title is confused `/PUT Update the job has just been create`. What does it mean?
- Seems that we're missing another test case like what happens if the job doesn't exist yet?

Reply:
Actually, we got another test case to create a new job above it, so I want it has to be clear that the job this test case modified is the one we have just created. In that case, can we just use "Update the job"?

Reply:
`so I want it has to be clear that the job this test case modified is the one we have just created` It's not a good idea. The unit test should be isolated and shouldn't depend on each other.
Please refactor it and add another test case for the case job doesn't exist yet.